packages/n3h-ipc/lib/server.js

```diff
@@ -174,7 +179,8 @@ class IpcServer extends AsyncClass {
       }
       this.emit('message', {
         name,
-        data
+        data,
+        from: zmqid
```

Comment:
Maybe we should just call this zmqid. I'm noticing in the code below, I'm getting confused about whether this is the `from` agentid

Reply:
oo, i like that :+1: